frontend/src/libs/constants/chat-mock-data.constant.ts

```diff
@@ -0,0 +1,13 @@
+import { ChatRole } from '#libs/enums/enums.js';
+import { type ChatMessage } from '#libs/types/types.js';
+
+const DOCTOR_FREUD_GREETING =
+  'Hello, I am Doctor Freud 👨‍⚕️. How can I help you?';
+
+const MOCK_MESSAGE: ChatMessage = {
+  id: crypto.randomUUID(),
```

Comment:
```suggestion
  id: generateUUID(),
```
Use the helper you have created